src/financial-overview/index.md

```diff
@@ -4,6 +4,8 @@ title: Financial Overview for Zinc Collective, LLC
 ---
 We publish our Profit and Loss and Balance Sheet reports on a quarterly and annual basis as part of our commitment to public and community governance and oversight.
 
+See the latest updates [here](https://opencollective.com/zinc-community/updates).
+
```

Comment:
It would be nice if I could get this link to open in a new tab. But right now it doesn't.

Reply:
I think you might be able to pass in the option to open in a new window/tab with:
```
[link](url){:target="_blank"}
```

Reply:
darn. it didn't work. had to go with straight HTML ce2b61de5c0bdd7ddf415a698ff72af05b7f109b